atd-vze/src/Components/GridTableSearch.js

```diff
@@ -0,0 +1,130 @@
+import React, { useState, useEffect } from "react";
+import { withApollo } from "react-apollo";
+import {
+  Col,
+  Button,
+  Form,
+  FormGroup,
+  Input,
+  InputGroup,
+  InputGroupAddon,
+  InputGroupButtonDropdown,
+  DropdownToggle,
+  DropdownItem,
+  DropdownMenu,
+  Alert,
+} from "reactstrap";
+
+// TODO add query operators to each field to better fit data types (_eq, etc.)?
+const GridTableSearch = ({ query, clearFilters, setSearchParameters }) => {
+  const [searchFieldValue, setSearchFieldValue] = useState("");
```

Comment:
I mentioned this to @mddilley too but I think there should be a default value for `searchFieldValue` and one of the dropdown options for search field is pre-selected. I'm familiar with the UI at this point and I always get bugged by the warning that I need to select a search value. My vote would be street name in this case but maybe for the abstraction it just needs to be the first item in the `query.searchableFields` array?